fossunited/fossunited/doctype/foss_event_schedule/foss_event_schedule.json

```diff
@@ -67,32 +60,51 @@
    "columns": 1,
    "fieldname": "linked_cfp",
    "fieldtype": "Link",
-   "in_list_view": 1,
    "label": "Linked CFP",
    "options": "FOSS Event CFP Submission"
   },
   {
    "fieldname": "section_break_ojro",
    "fieldtype": "Section Break"
   },
+  {
+   "description": "Format \n<pre>{\n  \"speakers\": [\n    {\n        \"name\": \"Test Name\",\n        \"designation\": \"Software Engineer\",\n        \"organization\": \"FOSS United\",\n        \"profile\": \"\",\n        \"image\": \"\",\n    },\n  ]\n}\n</pre>\n",
+   "fieldname": "speakers",
```

Comment:
I think Speakers could be better as added as a child table - but we don't have it in Frappe currently to have nested child tables.

Reply:
They wont be entering JSONs, Dashboard will have a UI, which will do that for them

Reply:
Agreed, just mentioned an observation